plugins/modules/identity/keycloak/keycloak_user_federation.py

```diff
@@ -83,9 +84,9 @@
     config:
         description:
             - Dict specifying the configuration options for the provider; the contents differ depending on
-              the value of I(provider_id). Examples are given below for C(ldap) and C(kerberos). It is easiest
-              to obtain valid config values by dumping an already-existing user federation configuration
-              through check-mode in the I(existing) field.
+              the value of I(provider_id). Examples are given below for C(ldap), C(kerberos) and C(sssd).
+              It is easiest to obtain valid config values by dumping an already-existing user federation
+              configuration through check-mode in the I(existing) field.
```

Comment:
```suggestion
              configuration through check-mode in the I(existing) field.
            - The value C(sssd) has been supported since community.general 4.2.0.
```